setup.py

```diff
@@ -148,6 +148,7 @@ def run(self):
             os.makedirs(CMAKE_BUILD_DIR)
 
         with cd(CMAKE_BUILD_DIR):
+            build_type = 'Release'
```

Comment:
I think cmake's default build type is not necessary "Release"?

Reply:
@raymondxyang `build_type` will be unconditionally appended to `cmake_args`

Reply:
Refer to https://stackoverflow.com/questions/24460486/cmake-build-type-not-being-used-in-cmakelists-txt

Reply:
After https://stackoverflow.com/questions/24460486/cmake-build-type-not-being-used-in-cmakelists-txt, there is no need apped build_type to cmake_args.